neurom/analysis/morphtree.py

```diff
@@ -264,6 +264,37 @@ def trunk_length(tree):
         return 0.0
 
 
+def trunk_vector(tree, soma):
+    '''Vector of trunk defined as (initial tree point - soma center)
+       of the tree.
+    '''
+    return mm.vector(tree.value, soma.center)
+
+
+def trunk_elevation(tree, soma):
+    '''Angle between x-axis and vector defined by (initial tree point - soma center)
+       on the x-y plane.
+    '''
+    vector = trunk_vector(tree, soma)
+
+    if np.linalg.norm(vector) != 0:
+        return np.arcsin(vector[COLS.Y] / np.linalg.norm(vector))
+    else:
+        raise ValueError("Vector between soma center and tree is [0, 0, 0]")
+
+
+def trunk_azimuth(tree, soma):
+    '''Angle between x-axis and vector defined by (initial tree point - soma center)
+       on the x-z plane.
+    '''
+    vector = trunk_vector(tree, soma)
+
+    if vector[COLS.X] != 0.0:
+        return np.arctan(vector[COLS.Z] / vector[COLS.X])
```

Comment:
Good point! I will use this instead:

http://docs.scipy.org/doc/numpy/reference/generated/numpy.arctan2.html